Xamarin.Essentials/Connectivity/Connectivity.ios.cs

```diff
@@ -13,6 +14,26 @@ static void StartListeners()
             listener.ReachabilityChanged += OnConnectivityChanged;
         }
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            var doubleSignalStrength = new NEHotspotNetwork().SignalStrength;
+            if (doubleSignalStrength < 0.01d)
+                return SignalStrength.None;
+            var signalstrength = (int)Math.Round(doubleSignalStrength * 3);
+            switch (signalstrength)
+            {
+                case 0:
+                case 1:
+                    return SignalStrength.Weak;
+                case 2:
+                    return SignalStrength.Fair;
+                case 3:
+                    return SignalStrength.Strong;
+                default:
+                    throw new InvalidOperationException("Exhaustive switch statement hit default value.");
```

Comment:
I agree with @mattleibow we should return unknown here.

```suggestion
                    return SignalStrength.Unknown;
```